app/static/js/controllers/clusterController.js

```diff
@@ -200,6 +200,11 @@ angular.module('uluwatuControllers').controller('clusterController', ['$scope',
                 result.blueprintId = parseFloat(result.blueprintId);
 
                 var existingCluster = $filter('filter')($rootScope.clusters, {id: result.id}, true)[0];
+                if (result !== undefined && result.id !== undefined) {
```

Comment:
I think it is not necessary to do a get request for getting the platformVariant field, it could be get from the posted Javascript object ($scope.cluster in our case).